crates/vm/levm/src/vm.rs

```diff
@@ -664,7 +672,7 @@ impl VM {
         let sender_account = self
             .cache
             .get_mut_account(current_call_frame.msg_sender)
-            .unwrap();
+            .ok_or(VMError::FatalUnwrap)?;
```

Comment:
Let's create a more explicit internal error for this, could be `InternalError::AccountNotFound`.